### PR TITLE
Migrates to yarn.lock

### PR DESCRIPTION
**Pull request by haishengwu-okta**

### CODE REVIEW

**Comment by haishengwu-okta** [comment]
@jmelberg-okta good catch!
pushed another update but the bacon scripts would be left out until https://oktainc.atlassian.net/browse/OKTA-190445 is solved.

File: README.md [hunk 1]
```diff
@@ -59,10 +59,10 @@ To install [@okta/okta-auth-js](https://www.npmjs.com/package/@okta/okta-auth-js
 
 ```bash
 # Run this command in your project root folder.
-npm install @okta/okta-auth-js --save
+yarn add @okta/okta-auth-js --save
```
Comment (jmelberg-okta) [inline README.md]: Lets also add the install command for `npm` too:

```bash
# yarn
yarn add --save @okta/okta-auth-js

# npm
npm install --save @okta/okta-auth-js
```

File: README.md [hunk 2]
```diff
@@ -59,10 +59,10 @@ To install [@okta/okta-auth-js](https://www.npmjs.com/package/@okta/okta-auth-js
...
 ```
 
-After running `npm install`, the minified auth client will be installed to `node_modules/@okta/okta-auth-js/dist`. You can copy the `dist` contents to a publicly hosted directory. However, if you're using a bundler like [Webpack](https://webpack.github.io/) or [Browserify](http://browserify.org/), you can simply import the module using CommonJS.
+After running `yarn add`, the minified auth client will be installed to `node_modules/@okta/okta-auth-js/dist`. You can copy the `dist` contents to a publicly hosted directory. However, if you're using a bundler like [Webpack](https://webpack.github.io/) or [Browserify](http://browserify.org/), you can simply import the module using CommonJS.
```
Comment (jmelberg-okta) [inline README.md]: We can probably make this more generic by saying:
"After running `yarn add`" ->  "After installing AuthJS"

File: README.md [hunk 3]
```diff
@@ -1616,21 +1616,19 @@ In most cases, you won't need to build the SDK from source. If you want to build
...
 git clone https://github.com/okta/okta-auth-js.git
 
-# Navigate into the new `okta-auth-js` filder, and install Okta node dependencies
+# Navigate into the new `okta-auth-js` filder
 cd okta-auth-js
-npm install
-
-# Build the SDK. The output will be under `dist`
-npm run build
+# Install Okta node dependencies and SDK will be built under `dist`
```
Comment (jmelberg-okta) [inline README.md]: nit: Add a newline between the command and comment:

```bash
cd okta-auth-js

# Install Okta...
```

**Pull request merged**